LEAF_Request_Portal/utils/db_procedures/data_stringReplace.php

```diff
@@ -0,0 +1,28 @@
+<?php
+ini_set('display_errors', 0); 
+
+
+define("LF", "\n");
+include '../db_mysql.php';
+include '../db_config.php';
+
+$debug = false;
+$db_config = new DB_Config();
+
+$db = new DB($db_config->dbHost, $db_config->dbUser, $db_config->dbPass, $db_config->dbName);
+
+// The find and replace strings.
+$find = "Database Error";
+$replace = "";
+$loop = mysql_query("
+SELECT
+concat('UPDATE ',table_schema,'.',table_name, ' SET ',column_name, '=replace(',column_name,', ''{$find}'', ''{$replace}'');') AS s
```

Comment:
```
''{$find}'', ''{$replace}'');'
```
These seem confusing, do you need this many single quotes?

Reply:
reference: https://talktopoint.com/replace-text-in-mysql-database/